src/services/WolfDen.Domain/Entity/StatusType.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class StatusType

```

Comment:
What does this table indicates, avoid using generic names
Use names that express the entities

Reply:
this table is about the statuses like present ,absent,late...
we have first given it as enum,but it was asked to change and was given in a seperate table